core/src/main/scala/filodb.core/downsample/DownsampleConfig.scala

```diff
@@ -34,6 +34,8 @@ final case class DownsampleConfig(config: Config) {
                 else Seq.empty
   val indexLocation = config.getOrElse[Option[String]]("index-location", None)
 
+  val maxRefreshHours = config.getOrElse("max-refresh-hours", Long.MaxValue)
```

Comment:
NIT: Should we add comment here on how this new variable `maxRefreshHours` will be used ? ( purely a nitpick comment since I am not sure what is the commenting convention we follow for this codebase )

Reply:
understood the usage later in the PR. thank you :)